### ISSUE TITLE
Can't update from Umbraco Forms 10.5.6 to Umbraco Forms 11.. 

### ISSUE THREAD

**Issue by mistyn8** [umbraco/Umbraco.Forms.Issues#1265]
It would seem that the latest migration present in 10.5.6 sets the migration key to `22df962a-ae26-4bdd-b8fd-0513a9c636bf` but that migration step doesn't exist in any version of umbraco forms 11. (10.5.6 being newer than any 11) and so the migration fails and the site fails to load.
`.To<EnsureIndexOnFormFolderKey>("22df962a-ae26-4bdd-b8fd-0513a9c636bf");`

I can see it's present in 12.2.4

I know I could go umb 10 straight to 13 (skipping between LTS).. but I've always done updates via the last release of interim majors.. for the reason mentioned in the docs.. 

> You can upgrade from Umbraco 10 to the latest version directly. If you choose to skip upgrading to versions 11 and 12, you will no longer receive warning messages for obsolete features. However, if you do skip these versions, any breaking changes will no longer compile

My reasoning is to catch the warnings for obsolete features rather than have to mend a broken site with no context.

I can see it is a tricky one.. as you are supporting the LTS 10, and with 11/12 being STS out of support, the LTS 13 and brand new 14 will be your focus..  (I'm also having to go to 11 first as that's the last release to support DoctypeGridEditor)

It's almost like you have to flag the v10 LTS patch as being outside the version flow of 11/12 but how you'd do that I'm not sure?

I did mange to work around it by setting the migration value in the db back to `.To<NoopMigration>("5d84fee1-388c-4e5f-b98c-1e66947278f1")` and the v11 instance is now working.

**Comment by AndyButland** [comment]
Yes, I see the issue.  Sounds like you have worked around it but just to make the method clearer for anyone else coming across this, once you have upgraded to 11 run this in the database:

```
UPDATE umbracoKeyValue
SET value = '5d84fee1-388c-4e5f-b98c-1e66947278f1' -- last migration state from Forms 11
WHERE [key] = 'Umbraco.Core.Upgrader.State+UmbracoForms'
```

As you say, that'll set the migration state back to the last known one in 11.  Once you upgrade to 12 or higher, it'll pick up from there, run the update that you have "missed" (but actually already have from 10)..

**Comment by Amalie-Wowern** [comment]
We’re doing a bit of cleanup and noticed this issue hasn’t had any activity for over a year, so we’ll close it for now.
If it’s still relevant, you’re welcome to open a new issue with the latest details.
